vmdb/app/models/ems_openstack_infra.rb

```diff
@@ -16,7 +16,7 @@ def supports_port?
   end
 
   def supported_auth_types
-    %w(default amqp)
+    %w(default amqp keypair)
```

Comment:
How about just `ssh_keypair`?  That says it's a keypair used specifically for ssh'ing into the system.